src/main/java/bio/terra/service/profile/ProfileService.java

```diff
@@ -207,11 +204,15 @@ public PolicyModel addProfilePolicyMember(String profileId,
                 policyName,
                 policyMember.getEmail());
         } catch (IamUnauthorizedException ex) {
-            if (ex.getCause() instanceof ApiException) {
-                ApiException samEx = (ApiException) ex.getCause();
-                if (samEx.getCode() == 403) {
-                    logger.warn("Ignoring not found exception", ex);
-                    return new PolicyModel();
+            // TODO: If we are not enforcing authorization then we allow there to be a missing profile
+            //  sam resource. Remove during clean up.
+            if (!applicationConfiguration.isEnforceBillingProfileAuthorization()) {
+                if (ex.getCause() instanceof ApiException) {
+                    ApiException samEx = (ApiException) ex.getCause();
+                    if (samEx.getCode() == 403) {
```

Comment:
do we have enums for these living anywhere?

Reply:
There is an enum, but for some crazy reason it is missing FORBIDDEN. I should add a comment.